spie-proceedings.csl

```diff
@@ -84,20 +84,6 @@
       </else>
     </choose>
   </macro>
-  <macro name="publisher">
-    <!-- Includes publisher place except for thesis -->
-    <choose>
-      <if type="thesis">
-        <text variable="publisher"/>
-      </if>
-      <else>
-        <group delimiter=", ">
-          <text variable="publisher"/>
-          <text variable="publisher-place"/>
-        </group>
-      </else>
-    </choose>
-  </macro>
```

Comment:
There are still a lot of other places in the style where the "publisher" macro is used, and needed, since http://spie.org/conferences-and-exhibitions/authors-and-presenters/format-your-manuscript-multimedia-files-and-references gives e.g. the following example:

>For a book: authors (last name first, then initials), [book title], publisher, city, page or chapter numbers (year). 
>
>[1] Booth, N. and Smith, A. S., [Infrared Detectors], Goodwin House Publishers, New York & Boston, 241-248 (1997).

So you can't just delete this macro.